.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Description
+
+### What works
+
+### What was not tested
+
+### Test
+Operating system (and linux kernel version): 
+Hardware (GPU/CPU):
```

Comment:
You should also use an itemization here, because both Operating system and Hardware are shown in a single line.